### DISCUSSION TITLE
Shared base link for editor & viewer [Aug 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6160] [Discord Threads]
> 🙋‍♂️ **productivix**  
## hi, do not seem to get a lot of answers here ... As the Github issues seems to be without reaction also , I publish here a bug in the users management : when we choose "activity share base enable", the link does NOT change when we toggle between "viewer" and "editor" : is that normal ? see attached  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1124353403783499879/viewer_editor_link.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Shared base link for editor & viewer permissions remains same. A base can be either shared with editor permissions or viewer permissions.    
####    
#### Is there any need for you to have two links - both active simultaneously?  
#  
> 🙋‍♂️ **productivix**  
#### hi, thank you , I do not need precise, I mention that it seems confuse or dangerous to have the same link for 2 different functions with 2 differents rights.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1124353404060315658)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
